pkg/config/dns-server/config.go

```diff
@@ -11,8 +11,6 @@ import (
 type DNSServerConfig struct {
 	// The domain that the server will resolve the services for
 	Domain string `yaml:"domain" envconfig:"kuma_dns_server_domain"`
-	// Port on which the server is exposed
```

Comment:
What happens if user passes a config with `port`? Does it silently accept the config? Maybe we can use [deprecate.go](https://github.com/kumahq/kuma/blob/master/pkg/config/app/kuma-cp/deprecate.go) to print a message in the Kuma CP logs that DNS server won't be started and please use builtin dns

Reply:
this also will work if user has `kuma_dns_server_port` env

Reply:
I'm unsure, I think if users were setting the port they were likely using the DNS server and maybe a fail fast would make a lot of sense because they should really migrate. WDYT?

Reply:
But even this fact won't force you to clean up your kuma-cp.config :) at the same time errors/warnings force you to remove unused fields from the configuration

Reply:
Ha ok I understand now. Let me try that

Reply:
Do we think it's actually that useful?

Reply:
Not really, I don’t mind having as it is, let’s just delete the field